app/assess/templates/macros/theme/question_above_answer_html.jinja2

```diff
@@ -0,0 +1,6 @@
+{% macro question_above_answer_html(meta) %}
+
+    <h2 class="govuk-heading-s"><strong> {{ meta.question }} </strong></h2>
+   {{ meta.answer }}
```

Comment:
Don't we need to do ` | safe` here since it's html we're rendering? without it I think it will be an escaped html string and not the actual elements 
```suggestion
   {{ meta.answer | safe }}
```

Reply:
I thought I added it. Thanks for spotting!!